File: src/main/java/org/c3lang/intellij/index/C3StringStubIndexExtension.java
package org.c3lang.intellij.index;

import com.intellij.psi.PsiElement;
import com.intellij.psi.stubs.StringStubIndexExtension;

public abstract class C3StringStubIndexExtension<T extends PsiElement> extends StringStubIndexExtension<T>
{
    @Override
    public int getVersion()
    {
        return 7;
    }
}

File: src/main/java/org/c3lang/intellij/index/C3StringStubIndexExtension.kt
package org.c3lang.intellij.index

import com.intellij.psi.PsiElement
import com.intellij.psi.stubs.StringStubIndexExtension

abstract class C3StringStubIndexExtension<T: PsiElement> : StringStubIndexExtension<T>() {
    override fun getVersion(): Int = 5
}